Google.Api.Generator.Tests/ProtoTests/Lro/Lro.proto

```diff
@@ -32,13 +32,12 @@ message Request {
   string name = 1;
 }
 
-option (google.api.resource_definition) = {
-  symbol: "Resource",
-  pattern: "items/{item_id}"
-};
-
 message ResourceRequest {
-  string name = 1 [(google.api.resource_reference) = "Resource"];
+  option (google.api.resource) = {
+    type: "lro.example.com/Resource"
```

Comment:
Any chance we could give it a more concrete name, such as "Book"? Things like `class ResourceName : IResourceName` end up suggesting a correlation that isn't really there.
As a reader, I want to see which parts of the generated names are influenced by what's in the proto and which are effectively hard-coded... something like "resource" could be either.
(I'm fine for that to be a change in a separate PR though.)

Reply:
OK, yes, I see what you mean.
As you say, this change will be better done in a future PR.